subsys/bluetooth/host/adv.c

```diff
@@ -459,8 +495,8 @@ static bool ad_is_limited(const struct bt_data *ad, size_t ad_len)
 static int le_adv_update(struct bt_le_ext_adv *adv,
 			 const struct bt_data *ad, size_t ad_len,
 			 const struct bt_data *sd, size_t sd_len,
-			 bool ext_adv, bool scannable, bool use_name,
-			 bool force_name_in_ad)
+			 bool ext_adv, bool scannable,
+			 enum adv_name_type use_name)
```

Comment:
`use_name` sounds like a boolean value. May I suggestion something like `name_type` instead?

Reply:
Thanks, that's better.